events-api-reference.yaml

```diff
@@ -0,0 +1,1070 @@
+swagger: '2.0'
+info:
+  title: I/O Management API
```

Comment:
`Adobe I/O Events API` would be more appropriate I guess

Reply:
Updated to Adobe I/O Events API